.github/workflows/action.yaml

```diff
@@ -0,0 +1,42 @@
+name: Checks
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
```

Comment:
We should add release jobs.
Probably with: https://goreleaser.com/ci/actions/ (or whatever)
But this doesn't have to happen until the branch for `mount` or something useful can be targeted for release builds.